src/styles/typography.js

```diff
@@ -6,33 +6,42 @@ const WEIGHTS = {
   extrabold: 800,
 };
 
+const FONT_FAMILY = '"Nordnet Sans Mono", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif';
+
+const commonFontStyles = {
+  fontFamily: FONT_FAMILY,
+  letterSpacing: 'normal'
+}
+
 export default function createTypography() {
   const ifSmallDevice = (smallFontSize, largeFontSize) => ({
-    fontFamily:
-      '"Nordnet Sans Mono", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif',
+    ...commonFontStyles,
     fontSize: smallFontSize,
-    letterSpacing: 'normal',
     [`@media only screen and (min-width: ${SMALL_DEVICE}px)`]: {
```

Comment:
SMALL_DEVICE is still `360` here, need to change to 480

Reply:
@shmidt-i fixed here https://github.com/nordnet/nordnet-ui-kit/pull/632